server/aws.js

```diff
@@ -0,0 +1,48 @@
+const AWS = require('aws-sdk');
+
+AWS.config.update({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
+
+const sqs = new AWS.SQS({ region: 'us-west-1' });
+
+const inventoryMessage = { topic: 'Inventory', payload: 'Test from Inventory' };
+const searchMessage = { topic: 'Inventory', payload: 'Test from Inventory' };
+const recommendationsMessage = { topic: 'Inventory', payload: 'Test from Inventory' };
+
+const sqsParamsInventory = {
```

Comment:
Suggestion: Repetition here can be reduced with a helper function that constructs this object using message and queue url.